modules/cli/src/main/scala/scala/cli/commands/ShebangOptions.scala

```diff
@@ -3,7 +3,9 @@ package scala.cli.commands
 import caseapp._
 
 @HelpMessage(
-  """|This command is an equivalent of `run`, but it changes the way how
+  """|Access to command line arguments in shebang scripts
+     |
+     |This command is an equivalent of `run`, but it changes the way how
```

Comment:
```suggestion
     |This command is equivalent to `run`, but it changes the way
```
(no need of "how" here, I think, and not 100% sure, but "equivalent to" sounds better to me)